waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/service/OidcLoginService.kt

```diff
@@ -0,0 +1,22 @@
+package id.walt.webwallet.service
+
+import com.auth0.jwk.JwkProvider
+import com.auth0.jwk.JwkProviderBuilder
+import id.walt.webwallet.service.WalletServiceManager.oidcConfig
+import java.net.URL
+import java.util.concurrent.TimeUnit
+
+object OidcLoginService {
```

Comment:
not sure it's a service, doesn't do anything but property initialization
maybe it can be renamed to smth. like OidcLoginService-/-Configuration or -Data